test/integration/targets/podman_image/aliases

```diff
@@ -1,4 +1,4 @@
-shippable/posix/group3
+unsupported
```

Comment:
Tests that are having issues should be marked `disabled` instead of `unsupported`.

Also, an issue in the appropriate project should be opened to track the disabled test:

https://github.com/ansible/ansible/projects?utf8=%E2%9C%93&query=is%3Aopen+broken

Reply:
More details in the docs here: https://docs.ansible.com/ansible/devel/dev_guide/testing/sanity/integration-aliases.html#disabled

Reply:
@mattclay Thanks for information. I will keep a note of this for future.

Reply:
Thanks @mattclay for the details.